Extension/src/LanguageServer/cppBuildTaskProvider.ts

```diff
@@ -393,6 +393,16 @@ class CustomBuildTaskTerminal implements Pseudoterminal {
         }
         if (this.options.cwd) {
             this.options.cwd = util.resolveVariables(this.options.cwd);
+        } else {
+            const editor: TextEditor | undefined = window.activeTextEditor;
+            let folder: WorkspaceFolder | undefined = editor ? workspace.getWorkspaceFolder(editor.document.uri) : undefined;
+            if (!folder && workspace.workspaceFolders) {
+                // TODO: Use the workspace folder for the tasks.json?
```

Comment:
Is there a way to get the folder from tasks.json if tasks.json exists?

Reply:
I couldn't find a way -- that's why I just added  TODO.